spec/unit/users_inspector_spec.rb

```diff
@@ -149,5 +149,27 @@
       names = description.users.map(&:name)
       expect(names).to eq(names.sort)
     end
+
+    it "can deal with invalid utf-8 characters in /etc/passwd" do
+      expect(system).to receive(:read_file).with("/etc/passwd").
+        and_return("tux:x:100:100:invalid\255char:/home/tux:/bin/bash\n")
+      expect(system).to receive(:read_file).with("/etc/shadow").and_return(nil)
+
+      expected = UsersScope.new([
+        User.new(
+              name: "tux",
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Indent the first parameter one step more than the start of the previous line.